iOS/Scenes/Manga/MangaViewController.swift

```diff
@@ -540,7 +540,29 @@ extension MangaViewController {
 
     private func makeMenu() async -> [UIMenuElement] {
         var menus = [UIMenu]()
-        var actions = [
+        var actions: [UIMenuElement] = [
+            // mark as read & mark as unread submenu
+            UIMenu(title: NSLocalizedString("MARK_ALL", comment: ""), image: UIImage(systemName: "pencil.tip.crop.circle"),children: [
```

Comment:
remove the image for this I think, and add a space after the comma
```suggestion
            UIMenu(title: NSLocalizedString("MARK_ALL", comment: ""), image: nil, children: [
```

Reply:
removed the icon from the menu